### PR TITLE
UCS NTP Server module

### PR DESCRIPTION
**Pull request by movinalot**
##### SUMMARY

UCS NTP Server management and test

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

ucs_ntp_server

##### ANSIBLE VERSION
```
ansible 2.7.0.dev0 (ucs_ntp_server 92d7a421a3) last updated 2018/07/05 14:39:28 (GMT -400)
  config file = /Users/jomcdono/.ansible.cfg
  configured module search path = [u'/Users/jomcdono/Documents/src/ucs/ansible/lib/ansible/modules/remote_management/ucs']
  ansible python module location = /Users/jomcdono/Documents/src/ucs/ansible/lib/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.15 (default, Jun 17 2018, 12:46:58) [GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by dsoper2 — APPROVED** [review]
shipit

File: lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  state:
+    description:
+    - If C(absent), will remove an ntp server.
+    - If C(present), will add or update an ntp server.
+    choices: [absent, present]
+    default: present
+
+  name:
+    description:
+    - The ntp server ip address or hostname.
```
Comment (dagwieers) [inline lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py]: I'd write this as `NTP server IP address or hostname`.

File: lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+- Examples can be used with the L(UCS Platform Emulator,https://communities.cisco.com/ucspe).
+options:
+  state:
+    description:
+    - If C(absent), will remove an ntp server.
+    - If C(present), will add or update an ntp server.
+    choices: [absent, present]
+    default: present
+
+  name:
```
Comment (dagwieers) [inline lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py]: I would prefer a more specific parameter name, like `ntp_server`, you can still keep `name` as alias.

File: lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    name: pool.ntp.org
+    description: External NTP Server by hostname
+    state: present
+
+- name: Remove ntp server
+  ucs_ntp_server:
+    hostname: 172.16.143.150
+    username: admin
+    password: password
+    state: absent
```
Comment (dagwieers) [inline lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py]: I would move the state one line below. First the selector, than the action, is what I usually do.

**Review by dagwieers — CHANGES_REQUESTED** [review]
LGTM, almost :-)

File: lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py (hunk 1 above)
Comment (movinalot) [reply lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py]: Change made

File: lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py (hunk 2 above)
Comment (movinalot) [reply lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py]: name changed to ntp_server, name kept as an alias for ntp_server

File: lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py (hunk 3 above)
Comment (movinalot) [reply lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py]: Changed

**Review by dsoper2 — APPROVED** [review]
shipit

**Review by dagwieers — APPROVED** [review]

**Pull request merged**